warehouse/router/router.go

```diff
@@ -711,14 +712,32 @@ func (r *Router) loadReloadableConfig(whName string) {
 	r.config.enableJitterForSyncs = r.conf.GetReloadableBoolVar(false, "Warehouse.enableJitterForSyncs")
 	r.config.warehouseSyncFreqIgnore = r.conf.GetReloadableBoolVar(false, "Warehouse.warehouseSyncFreqIgnore")
 	r.config.cronTrackerRetries = r.conf.GetReloadableInt64Var(5, 1, "Warehouse.cronTrackerRetries")
+	r.config.uploadBufferTimeInMin = r.conf.GetReloadableDurationVar(180, time.Minute, "Warehouse.uploadBufferTimeInMin")
```

Comment:
Is `uploadBufferTimeInMin` solely used for generating metrics and unrelated to the functioning of syncs ? If yes, then the variable name should reflect that

Reply:
When you say generating metrics, what do you mean? Currently it's being used for getting latest staging file in `2x`-`3x` time window.

Reply:
By "generating metrics" I meant that it is being used only in `warehouse_track_upload_missing` computation flow of the cron tracker but not in job syncs.

> Currently it's being used for getting latest staging file in 2x-3x time window.

But only in cron tracker flow, rt ?

Reply:
Do you mean something like `cronTrackerUploadBufferTimeInMin`?

Reply:
yes

Reply:
I am not sure that's a good idea. Since it's a generic configuration and not relevant to corn logic. It can be used in other places as well if required. For things like retries, it makes sense to name them as `cronTrackerRetries` but for uploadBufferTimeInMin, I am not sure.